docs/content/en/docs/core-components/modules.md

```diff
@@ -198,6 +199,58 @@ The result of the step is stored in the following output variables:
 }
 ```
 
+### Manual capability 
+This capability executes [manual Commands](https://docs.oasis-open.org/cacao/security-playbooks/v2.0/cs01/security-playbooks-v2.0-cs01.html#_Toc152256491) and provides them through the [SOARCA api](/docs/core-components/api-manual).
+
+
+<!-- The manual capability will allow an operator to interact with a playbook. It could allow one to perform a manual step that could not be automated, enter a variable to the playbook execution or a combination of these operations.
+
+The main way to interact with the manual step is through SOARCA's [manual api](/docs/core-components/api-manual). The manual step should provide a timeout SOARCA will by default use a timeout of 10 minutes. If a timeout occurs the step is considered as failed. -->
+
+
+
+
+
+#### Success and failure
+
+The manual step is considered successful if a response is made through the [manual api](/docs/core-components/api-manual). The manual command can specify a timeout but if none is specified SOARCA will use a default timeout of 10 minutes. If a timeout occurs the step is considered as failed.
```

Comment:
> "the step is considered as failed"

due to ambiguity of `success` and `failure` in CACAO V2, I would explicitly say that this is the choice we make to manage this event. So we either incur in workflow exception step if defined, or just fail the execution.